web/controllers/userController/index.ts

```diff
@@ -473,7 +473,7 @@ async function get(wix_id: string, person: Pupil | Student): Promise<ApiGetUser>
         apiResponse.university = person.university;
         apiResponse.state = person.state;
         apiResponse.lastUpdatedSettingsViaBlocker = moment(person.lastUpdatedSettingsViaBlocker).unix();
-
+        apiResponse.isOfficial = person.module != null && person.moduleHours != null;
```

Comment:
This can be weakened to `||` to prevent border cases due to, e.g., manual data editing.